build-in-container/mariner-docker-builder.sh

```diff
@@ -56,6 +56,15 @@ cleanup() {
     docker rmi -f $(docker images -aq --filter reference="mcr.microsoft.com/mariner-container-build")
 }
 
+create_custom_repo_file() {
```

Comment:
No real issues with the approach, but we can also just pass REPO_LIST="./path/to/repo1 ./path/to/repo2 ./path/to/repo3" to the tools and it will also handle managing multiple repo files. 

(I'm guessing this makes it easier to put the repo file into the container though?)

Reply:
Yes, can copy the custom.repo file(s) to /scripts and pass the path to tools. That way, the user can continue to edit the custom.repo file(s) and the tool would just pick up the edited version